jekyll/_cci2/customizing-ios.md

```diff
@@ -0,0 +1,429 @@
+3. Choose Source Control > Commit.
+4. Select the Shared Data folder.
+5. Enter your commit message in the text field.
+6. Select the "Push to remote" option (if your project is managed with Git).
+7. Click the Commit Files button.
+A new `.xcscheme` file is located in the
+`xcshareddata/xcschemes` folder under your Xcode project. 
+8. Commit this file to your git repository so that CircleCI can access it.
+
+Simple projects should run with minimal or no configuration.
+
+By default, CircleCI will:
+
+* **Build your project with Xcode 7.0 on OSX 10.11** - You can
+select a specific version of Xcode by specifying the major and minor version in
+a [config.yml file]( {{ site.baseurl }}/2.0/configuration-reference/) in the `.circleci` directory of your
+repository. 
+* **Install any Ruby gems specified in a Gemfile** - We will run `bundle
+  install` and cache the installed gems.
+* **Install any dependencies managed by CocoaPods** - If a Podfile is
+  present, we will run `pod install`, or `bundle exec pod install` if
+  both a Podfile and a Gemfile are present, and also cache the installed pods.
+* **Run the "test" build action for detected workspace (or project) and scheme
+  from the command line** - We will use `xctool` to build either the workspace
+  we find in your repo, or the project, if there is no workspace.
+
+
+### CocoaPods
+
+CircleCI will automatically detect if your project is using [CocoaPods](https://cocoapods.org)
+to manage dependencies. If you are using CocoaPods, then we recommend that you
+check your [Pods directory into source control](http://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control).
+This will ensure that you have a deterministic, reproducible build.
+
+If CircleCI finds a `Podfile` and the `Pods` directory is not present (or empty)
+ then we will run `pod install` to install the necessary dependencies in the
+`dependencies` step of your build.
+
+## Supported build and test tools
+
+CircleCI's automatic commands cover a lot of common test patterns, and you can customize your build
+as needed to satisfy almost any iOS build and test strategy.
+
+### XCTest-based tools
+In addition to standard `XCTestCase` tests, CircleCI will automatically run tests
+written in any other tool that builds on top of XCTest and is configured to run
+using the "test" build action. The following test tools are known to work well on CircleCI
+(though many others should work just fine):
+
+* [XCTest](https://developer.apple.com/library/ios/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/01-introduction.html)
+* [Kiwi](https://github.com/kiwi-bdd/Kiwi)
+* [KIF](https://github.com/kif-framework/KIF)
+
+### Other tools
+Popular iOS testing tools like [Appium](http://appium.io/) and [Frank](http://www.testingwithfrank.com/) should also
+work normally, though they will need to be installed and called using custom commands.
+
+## Code signing
```

Comment:
We will have a separate code signing doc. We should remove the contents of this section and instead link to the code signing doc. While the doc is not yet written, we should replace the contents with something like `coming soon`—current instructions don't work on 2.0.